docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -73,19 +137,52 @@ the ``copy`` module::
     ...
     '''
 
-Building & Testing
+Formatting options
 ''''''''''''''''''
+These formatting functions are ``U()`` for URLs, ``I()`` for option names, ``C()`` for files and option values, ``M()`` for module names. Module names should be specified as ``M(module)`` to create a
+link to the online documentation for that module.
+
+
+Example usage::
+
+    Or if not set the environment variable C(ACME_PASSWORD) will be used.
+    ...
+    Required if I(state=present)
+    ...
+    Required if I(state) is C(preset)
+    ...
+    Mutually exclusive with I(project_src) and I(files).
+    ...
+    See also M(win_copy) or M(win_template).
+    ...
+    See U(https://www.ansible.com/tower) for an overview.
+
+
+.. note::
 
-Put your completed module file into the 'library' directory and then
+  If you wish to refer a collection of modules use ``C(..)``, e.g. ``Refer to the C(win_*) modules.``
```

Comment:
If you wish to refer to a collection of modules, use ``C(..)``, e.g. ``Refer to the C(win_*) modules.``